src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1144,19 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
+     * @param prefix  Prefix to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+    	if (fullString.startsWith(prefix)){
+    		return fullString.replaceFirst(prefix, "");
```

Comment:
the indentation is off here. 
TIP: you can get Eclipse to set indentation to 4 spaces as required by our coding standard. please check how?